themes/ddbasic/sass/components/panel/class-panel.scss

```diff
@@ -391,7 +391,7 @@
 
 .pane-node-field-ding-library-lead,
 .pane-node-field-ding-library-body {
-  @include span-columns(5 of 8);
+  width: 100%;
```

Comment:
Shouldn't this be `@include span-columns(8 of 8);` instead of width?

Reply:
`@include span-columns(8); ` is  the same `width: 100%`

Reply:
Yes, but why change the logic of usage?